src/applications/disability-benefits/all-claims/pages/privateMedicalRecordsRelease.js

```diff
@@ -87,7 +90,27 @@ export const uiSchema = {
 export const schema = {
   type: 'object',
   properties: {
-    providerFacility: providerFacilities,
+    providerFacility: {
+      type: 'array',
+      minItems: 1,
+      maxItems: 100,
+      items: {
+        type: 'object',
+        required: [
+          'providerFacilityName',
+          'treatmentDateRange',
+          'providerFacilityAddress',
+        ],
+        properties: {
+          providerFacilityName,
+          treatmentDateRange: {
+            type: 'object',
```

Comment:
this isn't supposed to be an 'array'?

Reply:
It's not. The short story is that the UI isn't supposed to have the ability to add multiple treatmentDateRange's (yet). But the possibility to do it later didn't want to be closed off. So the Schema and API accept the array, with a little transformation.